homeassistant/components/automation/__init__.py

```diff
@@ -375,6 +381,63 @@ def device_state_attributes(self):
         return {CONF_ID: self._id}
 
 
+async def async_validate_config_item(hass, config, full_config):
```

Comment:
Let's move this into a new file `config.py`

Reply:
Fixed